src/sst/elements/vanadis/os/include/process.h

```diff
@@ -134,8 +138,8 @@ class ProcessInfo {
     }
 
     ProcessInfo( SST::Output* output, std::string checkpointDir,
-        MMU_Lib::MMU* mmu, PhysMemManager* physMemMgr, int node, unsigned pid, VanadisELFInfo* elfInfo, int debug_level, unsigned pageSize , uint64_t coreCount)
-        : m_mmu(mmu), m_physMemMgr(physMemMgr), m_pid(pid), m_pgid(pid), m_tid(pid), m_elfInfo(elfInfo), m_pageSize(pageSize), m_coreCount(coreCount)
+        MMU_Lib::MMU* mmu, PhysMemManager* physMemMgr, int node, unsigned pid, VanadisELFInfo* elfInfo, int debug_level, unsigned pageSize , uint64_t numLogicalCores)
```

Comment:
You declared numLogicalCores as uint32_t. It probably doesn't matter much if it's 32b or 64b, but be consistent.